network/network.go

```diff
@@ -175,7 +175,7 @@ func (c *Collector) Collect(ch chan<- prometheus.Metric) {
 
 	var wg sync.WaitGroup
 
-	pods, err := c.k8sClient.CoreV1().Pods(c.namespace).List(ctx, metav1.ListOptions{})
+	pods, err := c.k8sClient.CoreV1().Pods(c.namespace).List(ctx, metav1.ListOptions{LabelSelector: "app=net-exporter"})
```

Comment:
Is there a better way to do this than hard coding the app label here?

Reply:
By chance, is `c.service` the app name in this case?

Reply:
It's usually a good practice to put this string into a `const` at the top of a file at least.

Additionally we often use a file called `key.go` to collect these kinds of "magic" strings :)

Also: usually we declare a separate variable first e.g.:
```
opts := metav1.ListOptions{
    LabelSelector: "app=net-exporter"
}
```